src/components/add-form/index.js

```diff
@@ -3,48 +3,51 @@ import RadioInput from '../radio-input';
 import './styles.css';
 
 const AddForm = ({ formState, handleForm }) => (
-  <>
-    <label htmlFor="productName">
+  <form className="addform-container">
```

Comment:
Where you render `AddForm` it is nested within a `form` tag, and then in this component you have an other form tag. I would suggest to remove the tag where you render the `AddForm` and leave this component just like this.

Reply:
![image](https://user-images.githubusercontent.com/37032132/142733516-e57272ab-5c70-487e-919a-9cc047557a59.png)

Reply:
Yan about the welcome, that should be resolved with susana's PR.

Reply:
Done